flake.nix

```diff
@@ -450,7 +450,11 @@
         // {
           benchmark = {
             type = "app";
-            program = "${self.flake.${system}.packages."plutarch:bench:perf"}/bin/perf";
+            program = "${self.flake.${system}.packages."plutarch-benchmark:bench:perf"}/bin/perf";
+          };
+          perf-diff = {
```

Comment:
Can we call this `benchmark-diff` for consistency?

Reply:
Yeah, should the `perf` and `perf-diff` binaries be renamed too?

Reply:
It would be good for `perf` to become `benchmark` and for `perf-diff` to become `benchmark-diff` in all cases. Once you've pushed this I'll test the branch and then merge it.